### PR TITLE
Bump ts to 2.3, fixes

### PR DESCRIPTION
**Pull request by denar90**

### CODE REVIEW

File: lib/metrics.ts [hunk 1]
```diff
@@ -26,6 +28,8 @@ module.exports = {
     metricsIds.VC85,
     metricsIds.TTInonvis,
     metricsIds.TTInonvis5s,
+    metricsIds.EndOfATrace,
```
Comment (denar90) [inline lib/metrics.ts]: I found out that for some sites `eot` and `onload ` are shown on a chart...

**Comment by samccone** [comment]
👍 lgtm

**Comment by paulirish** [comment]
👍  on shipping 2.0.1 btw with this.

**Comment by paulirish** [comment]
Ah, you already did it! Ace.

**Pull request merged**